backend/public/js/utils.js

```diff
@@ -5,3 +5,48 @@ function copyAsIframe(e) {
                  `</iframe></p>`
     navigator.clipboard.writeText(iframe);
 }
+
+
+function validateUserDeletion(form) {
+    userEmail = form.userId.getAttribute('data-email');
+    if (confirm(`Do you really want to delete the user (${userEmail})?`)) {
+        let userId = form.userId.value;
+        $.ajax({
+            url: '/user/delete',
+            type: 'POST',
+            contentType: 'application/x-www-form-urlencoded',
```

Comment:
Why are we using `application/x-www-form-urlencoded` ?

Reply:
If we want to send simple text/ ASCII data, then x-www-form-urlencoded is used. This is the default. It might be skipped.